Resources/Prototypes/Recipes/Lathes/devices.yml

```diff
@@ -19,3 +19,16 @@
   materials:
     Steel: 200
     Plastic: 300
+
+- type: latheRecipe
+  id: FlashPayload
+  icon:
+    sprite: Objects/Devices/payload.rsi
+    state: payload-flash-armed
+  result: FlashPayload
+  completetime: 200
```

Comment:
lathes were recently updated so that the completetime is in seconds, so this should probably just be `completetime: 1`

Reply:
noted